intuita-webview/src/jobDiffView/App.tsx

```diff
@@ -28,8 +29,12 @@ export const App = () => {
 	}, []);
 
 	if (viewProps.kind === 'CODEMOD') {
-		// a placeholder for the codemod description (Markdown)
-		return null;
+		return viewProps.docs !== null ? (
```

Comment:
One idea about nullability that I have coined but probably haven't shared (:sweat_smile:) is to lock out of nullability as soon as possible.
In this case, `viewProps.docs` should not be null, if there's nothing to render, do not render anything (or display an info/error, if applicable).
The fewer nullables we have to handle, the better.